src/Mvc/Mvc.Testing/src/PACKAGE.md

```diff
@@ -0,0 +1,78 @@
+## About
+
+`Microsoft.AspNetCore.Mvc.Testing` provides support for writing integration tests for MVC applications.
```

Comment:
Maybe remove the MVC specifier? It's also useful for Minimal APIs.

Reply:
Yes, but it was created for MVC, and it lives on the MVC namespace.

Reply:
I've updated the wording to:

> `Microsoft.AspNetCore.Mvc.Testing` provides support for writing integration tests for ASP.NET Core apps using MVC or Minimal APIs.